style/ComponentNative/Workbench.mcss

```diff
@@ -1,6 +1,7 @@
 .baseViewport {
   position: absolute;
   outline: 1px solid #aaa;
+  user-select: none;
```

Comment:
This seems to fix the PC becoming selected problem for me, when I set the style manually in the debugger